docs/user-guide/03-convenience-features.md

```diff
@@ -383,6 +383,70 @@ int caller()
 }
 ```
 
+---
+layout: user-guide
+permalink: /user-guide/convenience-features-if-let
+---
+## `if_let` syntax
+Slang supports `if (let ...)` syntax to simplify the code when checking the type of a value. The syntax is similar to Rust's
```

Comment:
"when checking the type of a value" is not what it really is here.
should be "when working with an `Optional<T>` value".

Reply:
should be `if (let name = expr)`

Reply:
act